From 8a044ede1a8d14784fbe7bfd7f5c48f309821e71 Mon Sep 17 00:00:00 2001
From: Le Philousophe <lephilousophe@users.noreply.github.com>
Date: Sun, 30 Oct 2022 18:23:06 +0100
Subject: [PATCH] Don't try to guess package file to install

If package changes its version during build, the file will be wrong
---
 build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.sh b/build.sh
index 9bbe876..57c4d81 100755
--- a/build.sh
+++ b/build.sh
@@ -51,7 +51,7 @@ do
 
   if [ ! -z "$doinstall" ]; then
     echo "Installing $pkgdir"
-    psp-pacman -U --noconfirm "${pkgdir}/${pkgfile}" --overwrite '*'
+    psp-pacman -U --noconfirm "${pkgdir}"/*.pkg.tar.gz --overwrite '*'
   fi
 
 done
-- 
2.34.1

